web/src/main/java/org/cbioportal/weblegacy/mixin/StructuralVariantMixin.java

```diff
@@ -30,7 +30,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-package org.cbioportal.web.mixin;
+package org.cbioportal.weblegacy.mixin;
```

Comment:
mixin is relocated, but not registered with a CustomObjectMapper. We may need to configure an object mapper to be used with the weblegacy controllers